app/physicians/page.tsx

```diff
@@ -20,10 +24,19 @@ export async function generateMetadata(): Promise<Metadata> {
 
 export default async function Physicians() {
   const physicians = (await getPhysicians()) as unknown as PhysicianBioType[];
+  const page = (await getPageByType(
+    PAGE_TYPES.PHYSICIAN_LIST_PAGE,
+    4,
+  )) as unknown as PageType;
+  const sortedPhysicians: PhysicianPageSectionType[] =
```

Comment:
I think there is a way to clean this up further but I think this is OK for right now.